src/features/nft/nftModals/changeStatusModal.tsx

```diff
@@ -0,0 +1,85 @@
+import React from 'react'
+// Components
+import { Modal } from '../../../common/components/Modal'
+import { Button } from '../../../common/components/Buttons'
+import Select, { TOption } from '../../../common/components/Select/Select'
+import { Input } from '../../../common/components/Inputs'
+import DatePicker from '../../../common/components/DatePicker'
+import Link from '../../../common/components/Link'
+
+export type TChangeStatusModal = {
+  title?: string
+  isOpen: boolean
+  statusOptions: Array<TOption>
```

Comment:
Can we use `tatusOptions: TOption[]` instead of `statusOptions: Array<TOption>`? Please also update for below sections?